.github/workflows/ci.yml

```
@@ -20,87 +20,86 @@ jobs:
-  #         script: gradle wrapper; ./gradlew :react-native-usercentrics:connectedAndroidTest
+  test-ios:
+    needs: test-rn
+    runs-on: macos-13
+
+    steps:
+      - uses: maxim-lobanov/setup-xcode@v1
+        with:
+          xcode-version: '15.2'
+
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)